File: lib/models/season.dart
import 'package:udevs_video_player/models/movie.dart';

class Season {
  Season({
    required this.title,
    required this.movies,
  });

  String title;
  List<Movie> movies;

  Map<String, dynamic> toJson() {
    final map = <String, dynamic>{};
    map['title'] = title;
    map['movies'] = movies.map((v) => v.toJson()).toList();
    return map;
  }

  @override
  String toString() => 'Season{title: $title, movies: $movies}';
}

File: android/src/main/kotlin/uz/udevs/udevs_video_player/models/configuration/Season.kt
package uz.udevs.udevs_video_player.models.configuration

import com.google.gson.annotations.SerializedName
import java.io.Serializable

data class Season(
    @SerializedName("title")
    val title: String,
    @SerializedName("movies")
    val movies: List<Movie>,
) : Serializable